cromwell.example.backends/cromwell.examples.conf

```diff
@@ -501,6 +501,21 @@ services {
     #   prefix = "" # can be used to prefix all metrics with an api key for example
     #   flush-rate = 1 second # rate at which aggregated metrics will be sent to statsd
     # }
+
+    # Stackdriver- Send metrics to Google's monitoring API
+    # class = "cromwell.services.instrumentation.impl.stackdriver.StackdriverInstrumentationServiceActor"
+    # config {
```

Comment:
Now that I think of it, since one can only instantiate either StatsD or Stackdriver for instrumentation, I will switch `config.statsd` to simply `config`. How does that sound?

Reply:
Sounds good to me!